app/src/components/datasets/DatasetContentTabs/Evaluation/ShowEvalModal/ResultsTab.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Box,
+  Grid,
+  Heading,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+  useToken,
+} from "@chakra-ui/react";
+import chroma from "chroma-js";
+import React from "react";
+import { type RouterOutputs, api } from "~/utils/api";
+
+import { useMemo } from "react";
+import ColoredPercent from "~/components/ColoredPercent";
+import { useAppStore } from "~/state/store";
+import { getOutputTitle } from "../getOutputTitle";
+
+export const ResultsTab = () => {
+  const evalId = useAppStore((state) => state.evaluationsSlice.showEvalModalId);
+  const results = api.datasetEvals.results.useQuery(
+    { datasetEvalId: evalId as string },
```

Comment:
Should we take visible model ids into account here?

Reply:
In this case I don't think so -- the modal interface seems divorced enough from the outputs table that it would probably be more confusing.